XamlControlsGallery/ControlPagesSampleCode/NumberBox/NumberBoxSample2_xaml.txt

```diff
@@ -0,0 +1,7 @@
+﻿<muxc:NumberBox x:Name="NumberBoxSpinButtonPlacementExample"
+    Header="Enter an integer:" 
+    Value="1" 
+    SpinButtonPlacementMode="Compact" 
+    SmallChange="10"
+	LargeChange="100" >
+</muxc:NumberBox>
```

Comment:
This doesn't have @chingucoding's original suggestion:
> Nit: " />" instead of "</..>

And now the LargeChange got too indented :)

Reply:
The trouble here is tabs vs. spaces. The whitespace before the LargeChange is a tab, but should be spaces. @SavoySchuler please ensure that we're using spaces.